hubspot/client_test.go

```diff
@@ -54,6 +55,49 @@ var _ = Describe("Client", func() {
 
 	})
 
+	Describe("Email", func() {
+		var (
+			server *httptest.Server
+		)
+
+		BeforeEach(func() {
+			server = httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+				if strings.Contains(r.RequestURI, "/email/public/v1/singleEmail/send") {
```

Comment:
Suggest to add the defer GinkgoRecover so the test doesn't panic if assertion fails.
(we usually add that if you have an assertion that is running in another go routine 👍 
```suggestion
				defer GinkgoRecover()
				if strings.Contains(r.RequestURI, "/email/public/v1/singleEmail/send") {
```